editors/sc-ide/core/sc_server.hpp

```diff
@@ -163,9 +170,9 @@ private slots:
     QAction * mActions[ActionCount];
 
     VolumeWidget *mVolumeWidget;
-    QTimer mRecordTimer;
-    boost::chrono::system_clock::time_point mRecordTime;
+    int mRecordTime;
```

Comment:
I know, boost supplies types for that. I had many troubles with those types, so I just decided: it is just a display. We could call it ````mRecordSeconds```` for clarity?